enarx-keep/src/backend/kvm.rs

```diff
@@ -42,10 +46,27 @@ impl backend::Backend for Backend {
     }
 
     fn shim(&self) -> Result<PathBuf> {
-        unimplemented!()
+        let components = [
+            ".",
+            "enarx-keep-sev-shim",
+            "target",
+            "x86_64-unknown-linux-musl",
+            "debug",
+            "enarx-keep-sev-shim",
+        ];
+
+        let path = components.iter().fold(PathBuf::new(), |p, c| p.join(c));
+        Ok(path)
```

Comment:
We probably want this relative to `env!("CARGO_MANIFEST_DIR")` for now. Also, I'd prefer a long list of wrapped `.join()` calls because it is more obvious what is happening.